README.md

```diff
@@ -142,6 +142,18 @@ try {
 }
 ```
 
+##### Leeway
+
+to account for clock skew times between the signing and verifying servers, 
```

Comment:
Maybe reference the `firebase/php-jwt` docs: "It is recommended that this leeway should not be bigger than a few minutes."

Reply:
Fixed in 81e595b